client/src/actions/index.js

```diff
@@ -288,3 +291,12 @@ export const authenticateUser = history => {
 			);
 	};
 };
+
+export const upload = (data) => {
+	return dispatch => {
+		axios
+			.post(`${ROOT}/photos/upload`)
```

Comment:
I believe the photo route handler (for now) is `${ROOT}/users/upload` but Julian can clarify. He may end up changing it to the route you have here eventually but I'm not sure what he wants to do for the MVP.